examples/ecs-firelens/fluent-bit.conf

```diff
@@ -0,0 +1,41 @@
+[SERVICE]
+    Log_Level debug
+
+# input used for stdout application logs
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
```

Comment:
Adding the standard out input is not needed; FireLens will add that automatically (and will conflict with this input).

Reply:
removed